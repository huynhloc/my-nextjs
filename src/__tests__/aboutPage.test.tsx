import React from 'react';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import About from '../pages/about';

describe('About Page', () => {
  it('renders aboutpage unchanged', () => {
    const tree = renderer.create(<About />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders a heading', () => {
    render(<About />);

    const heading = screen.getByRole('heading');
    expect(heading).toHaveTextContent('About Page');

    // const heading = screen.getByRole('heading', {
    //   name: 'About Page',
    // });
    // expect(heading).toBeInTheDocument();
  });
});
