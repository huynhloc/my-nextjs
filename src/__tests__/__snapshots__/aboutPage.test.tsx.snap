// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page renders aboutpage unchanged 1`] = `
<div>
  <h1>
    About Page
  </h1>
</div>
`;
